contrib/completion/bash/docker

```diff
@@ -723,38 +725,38 @@ __docker_complete_log_options() {
 	local syslog_options="env labels syslog-address syslog-facility syslog-format syslog-tls-ca-cert syslog-tls-cert syslog-tls-key syslog-tls-skip-verify tag"
 	local splunk_options="env labels splunk-caname splunk-capath splunk-format splunk-gzip splunk-gzip-level splunk-index splunk-insecureskipverify splunk-source splunk-sourcetype splunk-token splunk-url splunk-verify-connection tag"
 
-	local all_options="$fluentd_options $gcplogs_options $gelf_options $journald_options $logentries_options $json_file_options $syslog_options $splunk_options"
+	local all_options="$common_options $fluentd_options $gcplogs_options $gelf_options $journald_options $logentries_options $json_file_options $syslog_options $splunk_options"
 
 	case $(__docker_value_of_option --log-driver) in
 		'')
 			COMPREPLY=( $( compgen -W "$all_options" -S = -- "$cur" ) )
 			;;
 		awslogs)
-			COMPREPLY=( $( compgen -W "$awslogs_options" -S = -- "$cur" ) )
+			COMPREPLY=( $( compgen -W "$common_options $awslogs_options" -S = -- "$cur" ) )
```

Comment:
The common options would end up multiple times in $all_options.
This is not a problem, though because bash completion automatically removes duplicates.